src/electionguard/serializable.py

```diff
@@ -48,11 +55,23 @@ def to_json_file(
         """
         write_json_file(self.to_json(strip_privates), file_name, file_path)
 
+    @classmethod
+    def from_json_file(cls, file_name: str, file_path: str = "") -> T:
```

Comment:
This could use just the `file_path` to reduce the length of this method. This concatenation could be moved out. I only kept it because the write had that but it could be said that neither need it.